lang/java/avro/src/main/java/org/apache/avro/util/Utf8.java

```diff
@@ -157,7 +157,7 @@ public int compareTo(Utf8 that) {
   }
 
   /** Gets the UTF-8 bytes for a String */
-  public static final byte[] getBytesFor(String str) {
+  public static byte[] getBytesFor(String str) {
```

Comment:
why is `final` removed here?

Reply:
`static` already implies that is class based so no need to make it `final` too.